src/components/CheckboxInput/CheckboxInput.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './CheckboxInput.scss';
+
+const CheckboxInput = ({
+  id,
+  className,
+  isChecked,
+  isDisabled,
+  onChange,
+  children,
+}) => {
+  const inputId = id || uuid();
+
+  const handleChange = e => {
+    onChange(e.target.checked);
+  };
+
+  return (
+    <div className={cx('checkboxInput', { isDisabled })}>
+      <input
+        id={inputId}
+        checked={isChecked}
+        disabled={isDisabled}
+        onChange={handleChange}
+        type="checkbox"
+        className={className}
+        style={{ width: '24px', height: '24px' }}
```

Comment:
Since we've got a scss file for this now can we add the styling there and use existing variables as opposed to inline styling? The height and width for this one should be `$size-spacing-2-xl` if I'm not mistaken.